### PR TITLE
Draft: Pr/fix ci

### PR DESCRIPTION
**Pull request by bd4**

### CODE REVIEW

File: include/sycl_ext_complex.hpp [hunk 1]
```diff
@@ -840,7 +840,7 @@ _SYCL_EXT_CPLX_INLINE_VISIBILITY
   typedef
       typename cplex::detail::__libcpp_complex_overload_traits<_Tp>::_ValueType
           _ValueType;
-  return sycl::atan2<_ValueType>(0, __re);
+  return sycl::atan2(static_cast<_ValueType>(0), __re);
```
Comment (TApplencourt) [inline include/sycl_ext_complex.hpp:843]: please undo

**Comment by TApplencourt** [comment]
Nice, the CI is now passing!

**Comment by bd4** [comment]
Note that is not running anything now, just building. But at least this way we can make a requirement, and can manually test that tests pass on GPU.

**Comment by jle-quel** [comment]
Shouldn't we use `clang-format version 17.0.0` ?

**Comment by bd4** [comment]
> Shouldn't we use `clang-format version 17.0.0` ?

I like 14 because it is available / default version on Ubuntu 22.04 - it is reasonably new but does not require a special install on most machines. In practice, 14 and 17 usually format the same way.

**Pull request merged**